integrationtests/npm_test.go

```diff
@@ -0,0 +1,190 @@
+package integrationtests
+
+import (
+	"flag"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/jfrog/build-info-go/build"
+	buildutils "github.com/jfrog/build-info-go/build/utils"
+	"github.com/jfrog/build-info-go/entities"
+	"github.com/jfrog/build-info-go/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+var runNpmIntegrationTests = flag.Bool("npmIntegration", false, "Run the integration tests (in addition to the unit tests)")
+var logger = utils.NewDefaultLogger(utils.DEBUG)
```

Comment:
The integration test produces a HUGH amount of output to the stdout
```suggestion
var logger = utils.NewDefaultLogger(utils.INFO)
```